care/emr/api/viewsets/report/report_upload.py

```
@@ -0,0 +1,365 @@
+
+
+class ReportUploadViewSet(
+    EMRCreateMixin, EMRRetrieveMixin, EMRUpdateMixin, EMRListMixin, EMRBaseViewSet
+):
+    database_model = ReportUpload
+    pydantic_model = ReportUploadCreateSpec
+    pydantic_read_model = ReportUploadListSpec
+    pydantic_update_model = ReportUploadUpdateSpec
+    pydantic_retrieve_model = ReportUploadRetrieveSpec
+
+    filter_backends = [DjangoFilterBackend, OrderingFilter]
```

Ruff (0.14.6)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)